collection-loader.js

```diff
@@ -17,8 +17,7 @@ function loadCollectionItems(client, collections) {
 // Ugly. This function updates all the items in place.
 // However, this is way more readable than a pure version
 function updateItemsInPlace(client, depth, items) {
-  const collections = items.filter(item => item.type == "collection");
-
+  const collections = items.filter(item => get(item, ["type"], "") === "collection");
```

Comment:
when would an item be null?

Reply:
Checking on what exactly happened. But I think this fix is a good idea to avoid any future mishaps.